### PR TITLE
BUGFIX: Avoid prepending base URI to absolute URIs

### PR DESCRIPTION
**Pull request by 2case**
Bugfix for #4

### CODE REVIEW

File: Classes/RedirectService.php [hunk 1]
```diff
@@ -84,20 +84,25 @@ protected function buildResponse(Request $httpRequest, RedirectInterface $redire
...
             $response->setHeaders(new Headers([
-                'Location' => $httpRequest->getBaseUri() . $redirect->getTargetUriPath(),
+                'Location' => $location,
                 'Cache-Control' => 'no-store, no-cache, must-revalidate, post-check=0, pre-check=0',
                 'Expires' => 'Sat, 26 Jul 1997 05:00:00 GMT'
             ]));
-        } elseif ($statusCode >= 400 && $statusCode <= 599) {
-            $exception = new Exception();
-            $exception->setStatusCode($statusCode);
-            throw $exception;
```
Comment (aertmann) [inline Classes/RedirectService.php]: why is this removed? maybe you had an outdated version of the file?

**Comment by 2case** [comment]
Hm, i don't know... i have to check it on monday at the office 

Greetings,
Tom.

**Comment by kdambekalns** [comment]
Which Monday did you want to check this? ;) Any chance you'll get around to that?

**Comment by 2case** [comment]
Was a mistake with removing this lines. Added them back.
